doc/source/rllib/doc_code/rlmodule_guide.py

```diff
@@ -399,3 +399,50 @@ def setup(self):
 
 module = spec.build()
 # __pass-custom-marlmodule-shared-enc-end__
+
```

Comment:
This addition solves a todo from the RL Modules doc to show checkpointing.
It's not very extensive, but I think we should include more info in a general section on checkpointing that includes S3 checkpointing, checkpointing with and without tune etc outside of the RL Modules guide.

Reply:
+1 on a separate user-guide on checkpointing and resuming experiments.

Reply:
Added to backlog https://github.com/ray-project/ray/issues/37515